.babelrc

```diff
@@ -1,10 +1,15 @@
 {
   "presets": [
     ["@babel/env", { "modules": false }],
-    "@babel/stage-3",
     "@babel/flow",
     "@babel/react"
   ],
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    ["@babel/plugin-proposal-class-properties", { "loose": false }],
+    "@babel/plugin-proposal-json-strings"
+  ],
```

Comment:
Does anything except class properties is used here?

Reply:
yes, and it works well

Reply:
[A description of the official document of stage-3](https://github.com/babel/babel/tree/master/packages/babel-preset-stage-3#babelpreset-stage-3)

Reply:
I know about stage-3. I mean does this package contain json strings, dynamic imports or import meta? If not, there's not need for these plugins.

Reply:
Thank you for your reply, I understand, and have submitted amendments.